packages/integration-test-app/windows/integrationtest/integrationtest.vcxproj

```diff
@@ -15,12 +15,15 @@
     <ApplicationTypeRevision>10.0</ApplicationTypeRevision>
     <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0.18362.0</WindowsTargetPlatformVersion>
     <WindowsTargetPlatformMinVersion>10.0.16299.0</WindowsTargetPlatformMinVersion>
-    <AppxPackageSigningEnabled>false</AppxPackageSigningEnabled>
+    <AppxPackageSigningEnabled>true</AppxPackageSigningEnabled>
+    <PackageCertificateThumbprint>A0127717C97521B4025737538269561980990B47</PackageCertificateThumbprint>
+    <PackageCertificateKeyFile>integrationtest_TemporaryKey.pfx</PackageCertificateKeyFile>
   </PropertyGroup>
```

Comment:
cc @chiaramooney @asklar @vmoroz 

This seems to be going directly against some of the work Chiara is doing, but I am having trouble seeing a way to avoid it. We want to be able to deploy a release AppX for this to test Hermes-specific release bits locally, and in PRs. Because PRs are untrusted, we cannot keep the cert a secret.

We could maybe keep it secret if we had it CI only, but I think Hermes coverage belongs in PR, and it is important to be able to deploy locally.

A self-signed cert still seems like the best solution for this from my perspective.

Reply:
deploying from layout in CI (without signing) is how we should do this. The CLI will do that today so you just need to set the property that says "don't try to package". Chiara will remember what the name of the prop is.

Reply:
to clarify, we should _not_ check in PFXs

Reply:
Lemme see if I can coax release into deploying from layout

Reply:
any reason why not run the test in debug instead? catch some of those asserts and aborts etc

Reply:
I think `<AppxBundle>Never</AppxBundle>` might be what I need. Testing that out.

Reply:
We run in both debug + release. The two end up being quite different. One goes through web debugger + package server, and the other is JSI + Hermes, + packaged bytecode.

Reply:
that should only control whether we do  a bundle or a loose package. A bundle is a metapackage of a set of archs/configs, so e.g. it can contain {debug, release} x {x64, x86, arm64}

Reply:
the property you want is GenerateAppxPackageOnBuild = False to prevent the build from generating an appx in the first place (and you'll need to figure out a way to trick the CLI into deploying from layout)